src/SensioLabs/Melody/Configuration/ConfigurationParser.php

```diff
@@ -12,7 +12,7 @@
 class ConfigurationParser
 {
     const PACKAGE_DELIMITER = ':';
-    const PACKAGE_REGEX = '/^(|[a-zA-Z0-9]([_.-]?[a-zA-Z0-9]+)*\\/[a-zA-Z0-9]([_.-]?[a-zA-Z0-9]+)*)$/';
+    const PACKAGE_REGEX = '/^((|[a-zA-Z0-9]([_.-]?[a-zA-Z0-9]+)*\\/[a-zA-Z0-9]([_.-]?[a-zA-Z0-9]+)*)|php|ext-[a-zA-Z0-9_.-]*)$/';
```

Comment:
After `ext-` it should contain at least one char `+` in the ext name instead of 0 `*`